### PR TITLE
Convert packaging_test_pull_request & handle new directory layout

### PR DESCRIPTION
**Pull request by ekohl**

### CODE REVIEW

File: puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/packaging_test_pull_request.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+      - string:
+          name: branch
+          default:
+          description: "Base branch to merge PR into"
+      - pr_parameters
+    scm:
+      - foreman-rpm-packaging-pr
+    triggers:
+      - github_pr:
+          context: 'foreman-packaging'
```
Comment (ehelms) [inline puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/packaging_test_pull_request.yaml]: Maybe context here would be better as a scratch-build or test-build? or package? something to reflect that its doing a test of scratch build

Comment (ekohl) [reply puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/packaging_test_pull_request.yaml]: For Debian it ends up on https://stagingdeb.theforeman.org/ and doesn't know the term scratch. Just packaging?

Comment (ehelms) [reply puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/packaging_test_pull_request.yaml]: Sure - packaging works for now. In the future I envision splitting deb from rpm to make this clearer on a given PR.

Comment (ehelms) [reply puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/packaging_test_pull_request.yaml]: ACK pending this little update

Comment (ekohl) [reply puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/packaging_test_pull_request.yaml]: updated

**Pull request merged**